### PR TITLE
Change version to v1.0RC in label for website menu

### PR DESCRIPTION
**Pull request by sarahmaddox**
Changed relevant version numbers to v1.0 in config, including changing the title of the version menu to **v1.0RC**.

Fixes https://github.com/kubeflow/website/issues/1587

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/website/1589)

### CODE REVIEW

**Comment by sarahmaddox** [comment]
Preview: https://deploy-preview-1589--competent-brattain-de2d6d.netlify.com/

/assign @jlewi

File: config.toml [hunk 1]
```diff
@@ -123,7 +123,7 @@ url_latest_version = "https://kubeflow.org/docs/"
 
 # A variable used in various docs to determine URLs for config files etc.
 # To find occurrences, search the repo for 'params "githubbranch"'.
-githubbranch = "v0.7-branch"
+githubbranch = "v1.0-branch"
```
Comment (jlewi) [inline config.toml]: it looks like this is referring to kubeflow/manfiests

I think some of the links need to be updated.

```
./content/docs/gke/customizing-gke.md:      * one for CPU only machines [here](https://github.com/kubeflow/kubeflow/blob/{{< params "githubbranch" >}}/scripts/gke/deployment_manager_configs/cluster.jinja#L109).
./content/docs/gke/customizing-gke.md:      * one for GPU machines [here](https://github.com/kubeflow/kubeflow/blob/{{< params "githubbranch" >}}/scripts/gke/deployment_manager_configs/cluster.jinja#L149).
./content/docs/gke/customizing-gke.md:  * When making changes to the node pools you also need to bump the pool-version [here](https://github.com/kubeflow/kubeflow/blob/{{< params "githubbranch" >}}/scripts/gke/deployment_manager_configs/cluster-kubeflow.yaml#L37) before you update the deployment
```

Those links on customizing GKE should now be
https://github.com/kubeflow/manifests/tree/v1.0-branch/gcp/deployment_manager_configs

As opposed to kubeflow/kubeflow

Comment (sarahmaddox) [reply config.toml]: Done.

**Comment by sarahmaddox** [comment]
Thanks @jlewi PTAL.

**Pull request merged**